index.html

```diff
@@ -4,11 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="./assets/css/styles.css">
+  <title>The Flower Pot</title>
 </head>
 
 <body>
-  
+  <header>
+    <h1>The Flower Pot</h1> 
+   <ul>
+     <li>
+       <a href="#AboutUs">About Us</a>
+     </li>
+     <li>
+       <a href="#Contact">Contact</a>
+     </li>
+     <li>
+       <a href="#Gallery">Gallery</a>
+     </li>
+
+   </ul>
+  </header>
+  <div class="Now">

```

Comment:
wrap all divs (cards) in a section tag with a class container this will make styling easier